cognite_toolkit/cdf_tk/load.py

```diff
@@ -226,6 +226,15 @@ def get_id(self, item: DataSet) -> str:
     def delete(self, ids: Sequence[str]) -> None:
         raise NotImplementedError("CDF does not support deleting data sets.")
 
+    def create(
+        self, items: Sequence[T_Resource], ToolGlobals: CDFToolConfig, drop: bool, filepath: Path
+    ) -> T_ResourceList | None:
+        try:
+            return self.api_class.create(items)
+
+        except CogniteDuplicatedError:
+            return None
```

Comment:
This is risky. For example, if you try to create 100 transformations and one of them already exists. All of them will fail, and they will silently not be created.

Reply:
No, it's overloaded create for datasets only. But we should then probably enforce one by one creation?

Reply:
You can catch the exception and create those that were not duplicates. I can push a commit if you want it.